rslabel_package/rslabel/ret.py

```diff
@@ -124,68 +165,79 @@ def app(args):
             print('robosub.eecs.wsu.edu:/data/vision/labeling/' + src_dir)
             sys.exit(-1)
 
-        # If the annotations already exist on the server, pull them down to
-        # figure out what new labels were added for stat tracking.
-        previous_annotations = []
-        with sftp.cd(src_dir):
-            if os.path.basename(args.annotations) in sftp.listdir():
-                with tempfile.NamedTemporaryFile() as tempf:
-                    sftp.get(os.path.basename(args.annotations), tempf.name)
-                    with open(tempf.name, 'r') as f:
-                        previous_annotations = json.load(f)
-
-        with open(args.annotations, 'r') as f:
-            new_annotations = json.load(f)
-
-        if len(previous_annotations) and len(new_annotations) != len(previous_annotations):
-            print('Provided annotation file and server annotation file differ.')
-            sys.exit(-1)
+            # If the annotations already exist on the server, pull them down to
+            # figure out what new labels were added for stat tracking.
+        if tar_name not in clarification_tars:
```

Comment:
You will need to calculate statistics for clarification tars as well. Figure out how to integrate this with the flow of the program. It shouldn't be too hard.

Reply:
Future!